includes/class-wc-stripe-database-cache.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+defined( 'ABSPATH' ) || exit; // block direct access.
+
+/**
+ * Class WC_Stripe_Database_Cache
+ */
+
+/**
+ * A class for caching data as an option in the database.
+ *
+ * Based on the WooCommerce Payments Database_Cache class implementation.
+ * https://github.com/Automattic/woocommerce-payments/blob/4b084af108cac9c6bd2467e52e5cdc3bc974a951/includes/class-database-cache.php
```

Comment:
Non-blocking nit: maybe this should use `@see` to be super-explicit.
```suggestion
 * @see https://github.com/Automattic/woocommerce-payments/blob/4b084af108cac9c6bd2467e52e5cdc3bc974a951/includes/class-database-cache.php
```

Reply:
I just added this in cc434a2c81227e102cce61a9df4a15cd82cf6830.